docker/compose_files/docker-compose.dev.yml

```diff
@@ -1,8 +1,11 @@
 version: '3.5'
 
-# Hotload the React code.
+# Hotload the React code and alembic version files
```

Comment:
Expand on comment:
```1) Hotload the React code for frontend development in dev mode without rebuilding static files (which takes a while).
2) Mount alembic version files so that we can create new database migrations (which need to written out of Docker into the `alembic` source directory).```